packages/smart-contracts/packages/staking/contracts/Staking.sol

```diff
@@ -274,13 +283,13 @@ contract Staking is
         uint48 distributionPercentage
     ) external onlyRole(DEFAULT_ADMIN_ROLE) mPoolExists(poolId) {
         // Checks: distribution percentage should not exceed 100%
-        if (distributionPercentage > PERCENTAGE_PRECISION) {
+        if (distributionPercentage > uint48(PERCENTAGE_PRECISION)) {

```

Comment:
If `PERCENTAGE_PRECISION` is updated to `uint32` and it is equal to  100%, it would make sense to change distributionPercentage to `uint32` as well

Reply:
in testing when we assign percentages, the assigned percentage does not fit in `uint32`, we need `uint48`